### PR TITLE
[AND-163] Add Firebase Remote Config

### PR DESCRIPTION
**Pull request by pedroagostinhoaptoide**
**What does this PR do?**

   - Adds firebase remote config dependency and a repository to use it and fetch config flags.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] See by commit.

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [AND-163](https://aptoide.atlassian.net/browse/AND-163)

**What are the relevant tickets?**

  Tickets related to this pull-request: [AND-163](https://aptoide.atlassian.net/browse/AND-163)

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[AND-163]: https://aptoide.atlassian.net/browse/AND-163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AND-163]: https://aptoide.atlassian.net/browse/AND-163?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app-games/src/main/java/com/aptoide/android/aptoidegames/AptoideApplication.kt [hunk 1]
```diff
@@ -10,6 +10,7 @@ import androidx.work.Configuration
 import androidx.work.Configuration.Provider
 import cm.aptoide.pt.feature_campaigns.AptoideMMPCampaign
 import cm.aptoide.pt.feature_categories.analytics.AptoideAnalyticsInfoProvider
+import cm.aptoide.pt.feature_flags.domain.FeatureFlags
```
Comment (G0retZ) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/AptoideApplication.kt:13]: Lets separate this file changes in a separate commit

**Review by G0retZ — CHANGES_REQUESTED** [review]

**Review by GuiFPires — APPROVED** [review]
